common/p_user.cpp

```diff
@@ -117,6 +117,17 @@ void P_ClearPlayerCards(player_t& p)
 		p.cards[i] = false;
 }
 
+/**
+ * @brief Clear all powerups from a player.
+ *
+ * @param p Player to clear.
+ */
+void P_ClearPlayerPowerups(player_t& p)
+{
+	for (size_t i = 0; i < NUMPOWERS; i++)
+		p.powers[i] = false;
```

Comment:
`powers` is not a boolean, you should probably prefer 0.

Reply:
I adapted with what existed already - will change it now.